tpu_inference/__init__.py

```diff
@@ -42,10 +42,15 @@
 else:
     # Either running on TPU or CPU
     try:
+        tpu_type = ti.get_tpu_type()
+        # For tpu7x, show chiplets_per_chip since each chip has 2 chiplets.
+        # Other TPU types don't have chiplets, so we don't show this info.
+        chiplets = f" | num_chiplets_per_chip={ti.get_num_chiplets_per_chip()}" if (
+            tpu_type and "tpu7x" in tpu_type.lower()) else ""
         logger.info(f"TPU info: node_name={ti.get_node_name()} | "
-                    f"tpu_type={ti.get_tpu_type()} | "
+                    f"tpu_type={tpu_type} | "
                     f"worker_id={ti.get_node_worker_id()} | "
-                    f"num_chips={ti.get_num_chips()} | "
+                    f"num_chips={ti.get_num_chips()}{chiplets} | "
```

Comment:
I think the better option is to just maintain logic as-is and just rename `num_chips` and `get_num_chips()` into `num_cores` and `get_num_cores()`?

Reply:
that sounds good to me! please update the PR as you've mentioned!